src/settings/MobileAppSettings.js

```diff
@@ -30,6 +30,7 @@ class MobileAppSettings extends Settings {
     const currentLanguage = this.get(SETTINGS_KEYS.CURRENT_LANGUAGE);
     setCurrentLanguage(currentLanguage);
     setCurrencyLocalisation(currentLanguage);
+    setDateLocale(currentLanguage);
```

Comment:
I think this is more setting "format" rather than locale. I think this has a bit of potential for confusion, e.g. between a currency format and an actual unit of currency, or a date format and a timezone. Maybe something like `setCurrencyLanguage`, `setDateLanguage`, or `setCurrencyFormat(currentLanguage)`, `setDateFormat(currentLanguage)`?

Reply:
yeah you're right - will fix soon